Yoast/wordpress-seo
# Make fully working notification for indexation

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* This moves the indexation from the somewhat hidden tools menu to a notification on the dashboard, plugins page, updates page and the Yoast pages.

## Test instructions

This PR can be tested by following these steps:

* Delete all your indexables by running `TRUNCATE TABLE wp_yoast_indexable`, or alternatively using the yoast test helper to delete all indexable tables.
* Visit both the dashboard, plugins page, updates page (e.g. http://basic.wordpress.test/wp-admin/update-core.php) and the Yoast pages. You should see a notification on those pages but not anywhere else.
* Run the indexation. You should now have entries in your indexables table and the warning should disappear.
* Repeat the first step.
* Dismiss the notification.
* It shouldn't show up anywhere else.

## UI changes

## Documentation

## Quality assurance
